api/internal/handler/schema/schema.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I think you can merge `plugin.go` to `schema.go`.

Reply:
oh, `plugin.go` not just expose schema, so we should keep it alone.